docs/platforms/apple/common/tracing/instrumentation/swiftui-instrumentation.mdx

```diff
@@ -80,3 +80,117 @@ var body: some View {
 This is the result
 
 ![Transaction with nested traced views](./img/nested-view-transaction.png)
+
+## Time to Initial Display and Time to Full Display
+
+<Alert>
+
+This feature is available since version 8.44.0 and only works for iOS and tvOS.
+
+</Alert>
+
+You can track time to initial display (TTID) and time to full display (TTFD) for your SwiftUI views to measure how long it takes for screens to render their first frame and then fully load all content.
+
+- **Time to initial display (TTID)**: Automatically tracked when the view appears on screen. The SDK sets the span operation to `ui.load.initial-display` and the span description to the view name followed by `initial display` - for example, `Content View Body initial display`.
```

Comment:
`m`: I think it would be great to mention exactly which hook we use to determine when the view appears on screen, which is `onAppear`, I believe https://developer.apple.com/documentation/SwiftUI/View/onAppear(perform:).